packages/common/src/visualizations/SqlRunnerResultsTransformer.ts

```diff
@@ -49,7 +49,7 @@ export type BarChartDisplay = {
     };
     yAxis?: {
         label?: string;
-        position?: 'left' | 'right';
+        position?: string;
```

Comment:
is this so we don't have an `echarts`-specific type?

Reply:
It was unnecessarily strict, it meant having to type a lot of other places. I'm experimenting with more relaxed typing, this led to much less boilerplate ❤️

Reply:
e.g. the value from a `Select` `onChange` is going to be a string, unless we do some type gymnastics